WORKING_GROUPS.md

```diff
@@ -40,11 +40,11 @@ for the Node.js project:
 Its responsibilites are:
```

Comment:
Can you add a comment here like:
```
<!--
This information should mostly mirror:
https://github.com/nodejs/inclusivity/blob/master/README.md#list-of-responsibilities
-->
```

Also, consider doing a PR to [inclusivity/README.md](https://github.com/nodejs/inclusivity/blob/master/README.md) with comments to help any possible future changes get propagated to the mirrors.